deploy/_doppler_deploy_common.sh

```
@@ -0,0 +1,91 @@
+# First, try Doppler normally
+if doppler run --config "$CONFIG" -- helmfile "$@" 2>/dev/null; then
+  exit 0
+fi
+
+# If that failed, resolve current path and check if we're on an external drive
+CURRENT_DIR="$(command -v realpath >/dev/null 2>&1 && realpath "$PWD" || pwd -P)"
+if [[ "$CURRENT_DIR" == /Volumes/* ]]; then
+  echo "⚠️  Detected external drive (resolved path: $CURRENT_DIR). Attempting alternative authentication..."
+
+  # Check for existing service token specific to this project & config
+  TOKEN_FILE="$HOME/.doppler-tokens/$(basename $(git rev-parse --show-toplevel 2>/dev/null || pwd))-$CONFIG"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)